spec/integration/init_web_spec.cr

```diff
@@ -13,6 +13,7 @@ describe "Initializing a new web project" do
       puts "Web app: Running integration spec. This might take awhile...".colorize(:yellow)
       should_run_successfully "lucky init test-project"
       FileUtils.cp("#{src_folder_in_docker}/spec/support/cat.gif", "#{temp_folder}/test-project/public/assets/images/")
+      File.delete("test-project/.env")
       compile_and_run_specs_on_test_project
       File.read("test-project/.travis.yml").should contain "postgresql"
       File.read("test-project/public/mix-manifest.json").should contain "images/cat.gif"
```

Comment:
Could  you add another `File.exists?` expectation for the env file like the spec below? That way we can be sure it gets created

Reply:
It already gets checked on line 21. Deleted on 16, created on 17, then checked for existence on 21.

Reply:
Another facepalm moment for me :) LGTM

Reply:
my github says all checks have failed... So not sure what's up with that 🤔